app/views/curation_concerns/base/_representative_media.html.erb

```diff
@@ -1,4 +1,4 @@
-<% if work.representative_id.present? && (fs = ::FileSet.load_instance_from_solr(work.representative_id)) %>
+<% if presenter.representative_id.present? && (fs = ::FileSet.load_instance_from_solr(presenter.representative_id)) %>
```

Comment:
For example, call `presenter.representative` and have it return the FileSet, loaded from solr, or nil.